services/tenants/transports/apikeys_http.go

```diff
@@ -0,0 +1,207 @@
+package tenantstransports
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strings"
+	"time"
+
+	"github.com/go-chi/chi/v5"
+
+	"sensorbucket.nl/sensorbucket/internal/httpfilter"
+	"sensorbucket.nl/sensorbucket/internal/pagination"
+	"sensorbucket.nl/sensorbucket/internal/web"
+	"sensorbucket.nl/sensorbucket/services/tenants/apikeys"
+)
+
+func NewAPIKeysHTTP(apiKeySvc apiKeyService, url string) *APIKeysHTTPTransport {
+	t := &APIKeysHTTPTransport{
+		router:    chi.NewRouter(),
+		apiKeySvc: apiKeySvc,
+		url:       url,
+	}
+	t.setupRoutes(t.router)
+	return t
+}
+
+type APIKeysHTTPTransport struct {
+	router    chi.Router
+	apiKeySvc apiKeyService
+	url       string
+}
+
+func (t *APIKeysHTTPTransport) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	t.router.ServeHTTP(w, r)
+}
+
+func (t *APIKeysHTTPTransport) setupRoutes(r chi.Router) {
+	r.Get("/api-keys", t.httpGetApiKeys())
+	r.Delete("/api-keys/revoke", t.httpRevokeApiKey())
+	r.Post("/api-keys/new", t.httpCreateApiKey())
+	r.Get("/api-keys/validate", t.httpValidateApiKey())
```

Comment:
For listing multiple entities use `list`, for single entities use `get`

What do you think about these endpoints?
`DELETE /api-keys/{id}`
`POST /api-keys` for creating new keys